geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala

```diff
@@ -180,89 +191,51 @@ class AccumuloDataStoreTest extends Specification with AccumuloDataStoreDefaults
       // check default
       val qpt = ds.getQueryPlan(query)
       qpt must haveSize(1)
-      qpt.head.table mustEqual ds.getTableName(sftName, Z3Table)
+      qpt.head.table mustEqual ds.getTableName(defaultTypeName, Z3Table)
       qpt.head.numThreads mustEqual 8
     }
 
     "allow users to call explainQuery" in {
-      val sftName = "explainQueryTest"
-      createSchema(sftName)
-      val query = new Query(sftName, Filter.INCLUDE)
-      val fr = ds.getFeatureReader(query, Transaction.AUTO_COMMIT)
-      fr must not beNull
+      val query = new Query(defaultTypeName, Filter.INCLUDE)
       val out = new ExplainString
-      ds.getQueryPlan(new Query(sftName, Filter.INCLUDE), explainer = out)
+      ds.getQueryPlan(new Query(defaultTypeName, Filter.INCLUDE), explainer = out)
       val explain = out.toString()
-      explain must startWith(s"Planning '$sftName'")
+      explain must startWith(s"Planning '$defaultTypeName'")
     }
 
     "allow secondary attribute indexes" >> {
-      val sftName = "AttributeIndexTest"
-      val sft = createSchema(sftName, "name:String:index=true,numattr:Integer,dtg:Date,*geom:Point:srid=4326")
-
-      val c = ds.connector
+      val sft = createNewSchema("name:String:index=true,numattr:Integer,dtg:Date,*geom:Point:srid=4326")
+      val sftName = sft.getTypeName
 
       "create all appropriate tables" >> {
-        "catalog table" >> { c.tableOperations().exists(defaultTable) must beTrue }
-        "st idx table" >> { c.tableOperations().exists(s"${defaultTable}_st_idx") must beTrue }
-        "records table" >> { c.tableOperations().exists(s"${defaultTable}_records") must beTrue }
-        "attr idx table" >> { c.tableOperations().exists(s"${defaultTable}_attr_idx") must beTrue }
+        val tables = GeoMesaTable.getTableNames(sft, ds)
+        tables must haveLength(4)
+        forall(Seq(Z2Table, Z3Table, AttributeTable, AttributeTable))(t => tables must contain(endWith(t.suffix)))
+        forall(tables)(t => ds.connector.tableOperations.exists(t) must beTrue)
       }
 
-      val pt = gf.createPoint(new Coordinate(0, 0))
+      val pt = WKTUtils.read("POINT (0 0)")
```

Comment:
just trying to clean up the test... I find using ECQL and WKT is easier to grok, and you don't need filter factories and geometry factories floating around.